src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -1456,6 +1481,14 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
   private def withAttributeNamespaceCheck[T](userInfo: UserInfo, hasAttributes: Attributable)(op: => Future[T]): Future[T] =
     withAttributeNamespaceCheck(userInfo, hasAttributes.attributes.keys)(op)
 
+  private def withAttributeNamespaceCheck[T](userInfo: UserInfo, methodConfiguration: MethodConfiguration)(op: => Future[T]): Future[T] = {
+    // TODO: this duplicates expression parsing, the canonical way to do this.  Use that instead?
+    // valid method configuration outputs are either in the format this.attrname or workspace.attrname
+    // invalid (unparseable) will be caught by expression parsing instead
+    val attrNames = methodConfiguration.outputs map { case (_, attr) => AttributeName.fromDelimitedName(attr.value.split('.').last) }
```

Comment:
It feels wrong to parse the outputs this way instead of using our expression parsing code, but I was unable to figure out how to make it work independently of DB object resolution.